design.md

```diff
@@ -288,12 +288,20 @@ inspiration from.
    schemas, need to be explicitly turned on in configuration, and follow
    all the other expectations we have around eventlogging data.
 
-5. *User consent / information UI*
+5. *User consent / information API and example UI*
 
    Every application collecting data should have a way to make it
    clear to the user what is being collected, and possibly ways
    to turn it off. We could possibly let admins configure opt-in /
-   opt-out options.
+   opt-out options. In some cases, we may wish to store consent information
+   separately from event data (this is commondly done in human subjects research),
```

Comment:
Typo: "commonly"

Reply:
Thanks @Zsailer. While I'm here - have you tried using the suggestion feature? You can create a suggestion by clicking the "+/- in a stack of 2 rectanglish things" icon.